### PR TITLE
Feature/adapt checkstyle and copyright

### PR DESCRIPTION
**Pull request by salmma**
#### Short Description
Adapt checkstyle and code style. Also update the copyrigth to a more general one.
#### Proposed Changes
  * update checkstyle.xml 
  * apply code style from checkstyle
  * update copyright in readme and all files
  * update list of contributors
  * update readme for checkstyle

### CODE REVIEW

File: NOTICE [hunk 1]
```diff
@@ -9,12 +9,21 @@ of the the Apache License, Version 2.0 which is available at https://www.apache.
...
+Copyright (c) 2020 Tobias Haller
+Copyright (c) 2020 Christoph Krieger
 Copyright (c) 2020 Tobias Mathony
+Copyright (c) 2020 Christian Müller
+Copyright (c) 2020 Tim Niederhausen
 Copyright (c) 2020 Marie Salm
+Copyright (c) 2020 Valentin Seifermann
 Copyright (c) 2020 Benjamin Weder
-Copyright (c) 2020 Manuela Weigold
+Copyright (c) 2020 Manuela Weigold
```
Comment (manuwei) [inline NOTICE:29]: Karoline Wild is missing

**Review by manuwei — COMMENTED** [review]
one contributer is missing (I added a comment for that), LGTM otherwise

File: NOTICE (hunk 1 above)
Comment (salmma) [reply NOTICE:29]: Thanks for the hint!

**Review by manuwei — APPROVED** [review]

**Review by wederbn — APPROVED** [review]
LGTM

Maybe we could integrate the checkstyle check into the CI?

**Pull request merged**